### ISSUE TITLE
The pivotea PKG is GREAT + a quick question: many rows and columns with: "NA"

### ISSUE THREAD

**Issue by sfd99** [matutosi/pivotea#1]
**Hi Toshikazu,**

Thanks for the new pivotea PKG! 
Really easy to use...
( version: 1.0.2 from CRAN, 
under latest Rstudio, Linux Ubuntu ).

When I run the starwars example line:
```
starwars |>
    pivot(row = "homeworld", col = "species", value = "name", split = "sex") 
```

 my results are correct but different 
 from your results, as shown in:
 
 https://matutosi.github.io/pivotea/
 
I get a lot of OK rows,
but...
also get  **many rows** and **many columns**
with entries are all "NAs",
specially for: 

   split: $male

**In Summary:**
my results for using split() with **starwars**,
look very different from your example results.
Many extra NA row and column values...

Please, let me know 
if I am missing something...
**SFd99
San Francisco.**

**Comment by matutosi** [comment]
Hi SFd99,

Thank you for your commments. 

In the function `pivot()`, `omit_na_cols()` and `omit_na_rows()` remove cols and rows that have only NAs.
Please try the codes below.

```
library(tidyverse)
library(pivotea)
tibble(a = rep(1:3), b = rep(NA, 3)) |>
  print() |>
  pivotea::omit_na_cols()
tibble(a = c(NA, 1), b = c(NA, 2)) |>
  print() |>
  pivotea::omit_na_rows()
```

If not working, please tell me the details.

**Comment by sfd99** [comment]
Ah, ok.

To try to get rid of
 all NA cols 
and all NA rows,
and  I tried:
```starwars  %>% 
      pivot(row = "homeworld", col = "species", value = "name", split = "sex")   %>% omit_na_cols() 
Error in `dplyr::select()`:
! `select()` doesn't handle lists
```
I also tried:
```
starwars  %>% 
      pivot(row = "homeworld", col = "species", value = "name", split = "sex")    %>%  omit_na_rows() 
Error in df[unlist(non_na), ] : incorrect number of dimensions
```

In _your starwars example output_,
no rows with all "NAs" are shown.
Your output results _have less rows and columns_.
That's great!.

But... in my Rstudio,
_with exactly the same starwars example_,
it shows some rows **with all NAs.**...

Maybe I'm not using the
  omit_na_cols() and
  omit_na_rows() functions correctly,
to reproduce your example output of:
```
 pivot(row = "homeworld", col = "species", value = "name", split = "sex")   ?...
 ```

**SFd99
San Francisco.**

**Comment by matutosi** [comment]
Need to use `purrr::map()`, because result of `pivot()` with split is a list.
```
starwars |>
  pivot(row = "homeworld", col = "species", value = "name", split = "sex") |>
  purrr::map(omit_na_rows) 
```

Although it looks all NA in the result of console, there mgiht be non-NA cell anywhere. 

```
result <- 
  starwars |>
  pivot(row = "homeworld", col = "species", value = "name", split = "sex")
nas <- # is NA
  result$female |>
  is.na()
apply(!nas, 1, sum) # no. of non-na cell in each row
apply(!nas, 2, sum) # no. of non-na cell in each column
```

**Comment by sfd99** [comment]
_Thanks Toshikazu!._
Your suggestion 
was really very helpful.

SFd99
San Francisco.
